client/signup/main.jsx

```diff
@@ -48,6 +48,11 @@ const Signup = React.createClass( {
 	displayName: 'Signup',
 
 	getInitialState() {
+		/**
+		 * Pass the redux store to SignupDependencyStore.
+		 */
```

Comment:
- My personal take, but comments this short are usually `// `-type comments
- The call that it seeks to comment is pretty self-explanatory, so I'd consider this comment superfluous